c2rust-analyze/src/main.rs

```diff
@@ -547,6 +609,7 @@ fn run(tcx: TyCtxt) {
         assert!(acx.local_tys.is_empty());
         acx.local_tys = IndexVec::with_capacity(mir.local_decls.len());
         for (local, decl) in mir.local_decls.iter_enumerated() {
+            // TODO: set PointerInfo::ANNOTATED for the parts of the type with user annotations
```

Comment:
can this even be gleaned from MIR? I thought type annotations were only visible at the HIR level